includes/html/modal/new_service.inc.php

```diff
@@ -13,6 +13,8 @@
 
 use LibreNMS\Authentication\LegacyAuth;
 
+include_once '/opt/librenms/includes/services.inc.php';
```

Comment:
Can you use Librenms install path variable. Because all install is not in /opt directory

Reply:
Bah, I forgot that was in there. Yep, I'll get that fixed.